internal/pkg/aws/apprunner/apprunner.go

```diff
@@ -213,6 +218,18 @@ func (a *AppRunner) WaitForOperation(operationId, svcARN string) error {
 	}
 }
 
+// PrivateURL returns the url associated with a VPC Ingress Connection.
+func (a *AppRunner) PrivateURL(ctx context.Context, vicArn string) (string, error) {
```

Comment:
nit: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
```suggestion
func (a *AppRunner) PrivateURL(ctx context.Context, vicARN string) (string, error) {
```

Reply:
going back and forth between cfn and go is hard 😓